docs/excel/add-properties-to-basic-cell-values.md

```diff
@@ -0,0 +1,398 @@
+    const range = sheet.getRange("A1");
+    range.valuesAsJson = scissors;
+    await context.sync();
+  });
+}
+```
+
+The following image shows how the string and its properties appear to the user as a custom card layout.
+
+:::image type="content" source="../images/data-type-basic-card-layout.png" alt-text="Data card layout image for scissors showing Quantity and price section, and Additional information section which is collapsed.":::
+
+## Nested data types
+
+You can nest data types in a cell value, such as additional entity values, as well as **strings**, **doubles**, and **Booleans**. The following code sample shows how to create a cell value that represents the charge status on a computer battery. It contains a nested entity value that describes the computer properties for power consumption and charging status. The computer entity value also contains a nested string value that describes the computer’s power plan.
+
+```javascript
+async function createNumberWithNestedEntity() {
+  await Excel.run(async (context) => {
+    const sheet = context.workbook.worksheets.getActiveWorksheet();
+    const range = sheet.getRange("A1");
+    range.valuesAsJson = [
+      [
+        {
+          type: Excel.CellValueType.double,
+          basicType: Excel.RangeValueType.double,
+          layouts: {
+            compact: {
+              icon: "Battery10"
+            }
+          },
+          basicValue: 0.7,
+          numberFormat: "00%",
+          properties: {
+            Computer: {
+              type: Excel.CellValueType.entity,
+              text: "Laptop",
+              properties: {
+                "Power Consumption": {
+                  type: Excel.CellValueType.double,
+                  basicType: Excel.RangeValueType.double,
+                  basicValue: 0.25,
+                  numberFormat: "00%",
+                  layouts: {
+                    compact: {
+                      icon: "Power"
+                    }
+                  },
+                  properties: {
+                    plan: {
+                      type: Excel.CellValueType.string,
+                      basicType: Excel.RangeValueType.string,
+                      basicValue: "Balanced"
+                    }
+                  }
+                },
+                Charging: {
+                  type: Excel.CellValueType.boolean,
+                  basicType: Excel.RangeValueType.boolean,
+                  basicValue: true
+                }
+              }
+            }
+          }
+        }
+      ]
+    ];
+    await context.sync();
+  });
+}
+```
+
+The following image shows the number value and the data type card for the nested Laptop entity.
+
+:::image type="content" source="../images/data-type-basic-nested-entities.png" alt-text="Cell value in Excel showing battery charge at 70% and the data type card showing the nested laptop entity with charging and power consuption property values.":::
+
+## Attribution
```

Comment:
For this Attribution content -- maybe we create a new article called **Data types attribution** and describe how to do attributions for all the types? I think the similarity with this [Card data attribution](https://learn.microsoft.com/en-us/office/dev/add-ins/excel/excel-data-types-entity-card#card-data-attribution) section could be confusing